docker-compose.yml

```diff
@@ -242,6 +244,26 @@ services:
       docktail.service.service-port: 443
       docktail.service.service-protocol: https
 
+  # https://github.com/Dictionarry-Hub/profilarr
+  # https://dictionarry.dev/profilarr-setup/installation
+  profilarr:
+    <<: *s6-service
```

Comment:
Profilarr is not s6-based, so we don't want to use these two related anchors.

Though it would be nice to have optional support for the `DISABLE` env var by replacing the default [entrypoint](https://github.com/Dictionarry-Hub/profilarr/blob/3a0deb16fa1c278e4b0f4020afe1e975327d5640/Dockerfile#L20) with a custom command.